packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -396,6 +396,42 @@ def self.set_imagemanager_search_path(target_installation_result)
         ])
     end
 
+    def self.get_plist_paths_from(user_project)
+        info_plists = user_project
+          .files
+          .select { |p|
+            p.name&.end_with?('Info.plist')
```

Comment:
`Info.plist` files should be discovered by tracing `PBXProject > PBXNativeTarget > XCConfigurationList > XCBuildConfiguration > buildSettings.INFOPLIST_FILE`.